### PR TITLE
Fixed a bunch of linting issues in components

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Some of the linting issues were existing issues, some are new ones due to the changes made to the linting ruleset for the frontend.

## QA Instructions, Screenshots, Recordings

N/A

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Actors from some show saying "It comes with a lint brush

### CODE REVIEW

File: app/javascript/chat/content.jsx [hunk 1]
```diff
@@ -5,59 +5,6 @@ import ChannelRequest from './channelRequest';
 import RequestManager from './requestManager';
 import ChatChannelSettings from './ChatChannelSettings/ChatChannelSettings';
 
-function display(resource) {
```
Comment (rhymes) [inline app/javascript/chat/content.jsx:8]: not used?

Comment (nickytonline) [reply app/javascript/chat/content.jsx:8]: Correct

File: app/javascript/Snackbar/SnackbarItem.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
-import { Button } from '@crayons';
 import { defaultChildrenPropTypes } from '../common-prop-types';
+import { Button } from '@crayons';
```
Comment (nickytonline) [inline app/javascript/Snackbar/SnackbarItem.jsx:4]: This is an automatically fixed lint rule, so as mentioned in #8880, this causes no burden on a contributor.

File: app/javascript/article-form/components/EditorActions.jsx [hunk 3]
```diff
@@ -77,9 +77,7 @@ export class EditorActions extends Component {
             className="mr-2 whitespace-nowrap"
             onClick={onSaveDraft}
           >
-            Save 
-            {' '}
-            <span className="hidden s:inline">draft</span>
+            Save <span className="hidden s:inline">draft</span>
```
Comment (nickytonline) [inline app/javascript/article-form/components/EditorActions.jsx:80]: These types of changes are prettier cleanup.

**Pull request merged**